api/apps/api/src/modules/projects/edit-guard/marxan-edit-guard.service.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { Test } from '@nestjs/testing';
+import { EditGuard } from '@marxan-api/modules/projects/edit-guard/edit-guard.service';
+import { ProjectChecker } from '@marxan-api/modules/projects/project-checker/project-checker.service';
+import { ProjectCheckerFake } from '../../../../test/utils/project-checker.service-fake';
+import { v4 } from 'uuid';
+import { FixtureType } from '@marxan/utils/tests/fixture-type';
+import { MarxanEditGuard } from '@marxan-api/modules/projects/edit-guard/marxan-edit-guard.service';
+
+describe('MarxanEditGuard', () => {
+  let fixtures: FixtureType<typeof getFixtures>;
+
+  beforeEach(async () => {
+    fixtures = await getFixtures();
+  });
+
+  it(`throws an exception if the given project has an ongoing export`, async () => {
+    const projectId = fixtures.GivenProjectWasCreated();
+
+    await fixtures.WhenExportIsRequested(projectId);
+
+    await fixtures.ThenAnExceptionIsThrown(projectId);
+  });
+
+  it(`does nothing if the given project does not have an ongoing export`, async () => {
+    const projectId = fixtures.GivenProjectWasCreated();
+
+    await fixtures.ThenNothingHappens(projectId);
```

Comment:
as above, I'd suggest to try to clarify the naming of this, if not fully breaking it down (which would make even more sense here where we're outright missing a `when` step), e.g.
```suggestion
    await fixtures.NoExceptionIsThrownIfCheckingWhetherProjectCanBeEdited(projectId);
```

as it is now, although the code is testing the right behaviour, our description reads "given a project was created, then nothing happens", which at to someone reading it at a glance may sound confusing